scripts/clients.py

```diff
@@ -121,9 +121,8 @@ def create_user_on_appserver(self, user: dict) -> dict:
                 print(f"The user {user} already exists")
                 return requests.get(f"{user_endpoint}/{user['subjectId']}").json()
             else:
-                print(
-                    f"Error making post request to create User: {user}, {response.status_code}, {response.url}, {response.content}, {response.headers}")
-                exit(1)
+                # print(f"Error making post request to create User: {user}, {response.status_code}, {response.url}, {response.content}, {response.headers}")
```

Comment:
sorry, my fault, but can you please remove these print comments from all over the file? Thanks.